components/processing/terraform/main.tf

```diff
@@ -98,7 +101,7 @@ module "gcloud" {
   version = "~> 3.4"
 
   create_cmd_entrypoint = "gcloud"
-  create_cmd_body       = "builds submit --region ${var.region} --project ${var.project_id} --config \"${local_file.cloudbuild_cloud_run.filename}\" \"${path.module}/../../../..\""
+  create_cmd_body       = "builds submit --region ${var.region} --project ${var.project_id} --config \"${local_file.cloudbuild_cloud_run.filename}\" \"${path.module}/../../..\""
```

Comment:
I believe Anurhada's request was to separate the build/deployment related "resources" to a different file (like `build.tf`) so it's easier to users to remove that in favor of a proper CI/CD pipeline.

Reply:
break out cloud build yaml and cloud build resource into `build.tf`